invokeai/app/api/models/images.py

```diff
@@ -32,3 +32,10 @@ class ProgressImage(BaseModel):
     width: int = Field(description="The effective width of the image in pixels")
     height: int = Field(description="The effective height of the image in pixels")
     dataURL: str = Field(description="The image data as a b64 data URL")
+
+
+class NonNullableImageField(BaseModel):
```

Comment:
See the comment about a delete body not being a good idea. You won't need this class without that method.